packages/commercetools/composables/src/getters/cartGetters.ts

```diff
@@ -72,13 +74,37 @@ export const getCartTotalItems = (cart: Cart): number => {
 
 export const getFormattedPrice = (price: number) => price as any as string;
 
-export const getCoupons = (cart: Cart): AgnosticCoupon[] => {
+export const getCartCoupons = (cart: Cart): AgnosticCoupon[] => {
   return getCouponsFromCart(cart);
 };
 
-// eslint-disable-next-line
-export const getDiscounts = (cart: Cart): AgnosticDiscount[] => {
-  return [];
+export const getCartDiscounts = (cart: Cart): AgnosticDiscount[] => {
+  const lineItems = getCartItems(cart);
+  return lineItems.reduce((reducedDiscounts, lineItem) => {
+    const discounts = lineItem.discountedPricePerQuantity;
+    const discountsLength = discounts.length;
```

Comment:
You can instead store a last discount item in this variable
```suggestion
    const lastDiscountItemIndex = discounts.length - 1;
    
    discounts[lastDiscountItemIndex]?.discountedPrice.includedDiscounts ....
```